src/idm/entity.js

```diff
@@ -0,0 +1,175 @@
+        * @param {String} entityType - type of entity
+        * @param {object} entity - An object containing the entity
+        * @fulfil {Object} entity created
+        * @returns {Promise}
+        * @example
+        * agile.idm.entity.createEntity('1','/sensor',{"name":"entity's name"}).then(function(result) {
+        *   console.log('entity created!'+result);
+        * });
+        **/
+        createEntity: (entity_id, entity_type, entity) => axios({
+          method: 'POST',
+          url: `${base}/api/v1/entity/${entity_type}/${entity_id}`,
+          headers: {"Authorization" : `bearer ${token}`},
+          data: entity
+        })
+        .then(res => (res.data))
+        .catch(errorHandler),
+    /**
+    * @summary Delete entity
+    * @name deleteEntity
+    * @public
+    * @function
+    * @memberof agile.idm.entity
+    * @param {String} entityId - id of entity
+    * @param {String} entityType - type of entity
+    * @fulfil {Undefined}
+    * @returns {Promise}
+    * @example
+    * agile.idm.entity.deleteEntity('1','/sensor').then(function() {
+    *   console.log('group removed!');
+    * });
+    **/
+    deleteEntity: (entity_id, entity_type) => axios({
+      method: 'DELETE',
+      url: `${base}/api/v1/entity/${entity_type}/${entity_id}`,
+      headers: {"Authorization" : `bearer ${token}`}
+    })
+    .then(res => (res.data))
+    .catch(errorHandler),
+    /**
+    * @summary Set Entity's attribute
+    * @name setEntityAttribute
+    * @public
+    * @function
+    * @memberof agile.idm.entity
+    * @param {String} entityId - id of entity
+    * @param {String} entityType - type of entity
+    * @param {String} attributeName- name of the attribute
+    * @param {Object|String} attribute - An object or a String containing the entity's attribute value
+    * @fulfil {Object} entity created
+    * @returns {Promise}
+    * @example
+    * agile.idm.entity.setEntityAttribute('1','/sensor',"credentials",{"dropbox":"entity's credentials for drop"}).then(function(result) {
+    *   console.log('entity created!'+result);
+    * });
+    **/
+    setEntityAttribute: (entity_id, entity_type, attribute_type, attribute_value) => axios({
+      method: 'PUT',
+      url: `${base}/api/v1/entity/${entity_type}/${entity_id}/attribute/${attribute_type}/`,
+      headers: {"Authorization" : `bearer ${token}`},
+      data: { "value" : attribute_value}
+    })
+    .then(res => (res.data))
+    .catch(errorHandler),
+    /**
+    * @summary Delete Entity's attribute
+    * @name deleteEntityAttribute
+    * @public
+    * @function
+    * @memberof agile.idm.entity
+    * @param {String} entityId - id of entity
+    * @param {String} entityType - type of entity
+    * @param {String} attributeName- name of the attribute
+    * @fulfil {Object} entity updated entity
+    * @returns {Promise}
+    * @example
+    * agile.idm.entity.deleteEntityAttribute('1','/sensor',"credentials").then(function(result) {
```

Comment:
I think you mean npm run lint right? I have tried to install the dependencies with --only=dev and then install. Then I get UNMET PEER DEPENDENCY eslint-config-standard@^6.0.0 if I do npm list.
Also, when I run npm run lint I get this: 
```
> agile-sdk@0.4.0 lint /home/dp/git/agile-sdk
> eslint src/**

Cannot find module 'eslint-config-standard'
Referenced from: semistandard
Referenced from: /home/dp/git/agile-sdk/.eslintrc
Error: Cannot find module 'eslint-config-standard'
Referenced from: semistandard
Referenced from: /home/dp/git/agile-sdk/.eslintrc
    at ModuleResolver.resolve (/home/dp/git/agile-sdk/node_modules/eslint/lib/util/module-resolver.js:74:19)
    at resolve (/home/dp/git/agile-sdk/node_modules/eslint/lib/config/config-file.js:478:25)
    at load (/home/dp/git/agile-sdk/node_modules/eslint/lib/config/config-file.js:495:26)
    at configExtends.reduceRight (/home/dp/git/agile-sdk/node_modules/eslint/lib/config/config-file.js:391:36)
    at Array.reduceRight (native)
    at applyExtends (/home/dp/git/agile-sdk/node_modules/eslint/lib/config/config-file.js:362:28)
    at load (/home/dp/git/agile-sdk/node_modules/eslint/lib/config/config-file.js:529:22)
    at configExtends.reduceRight (/home/dp/git/agile-sdk/node_modules/eslint/lib/config/config-file.js:391:36)
    at Array.reduceRight (native)
    at applyExtends (/home/dp/git/agile-sdk/node_modules/eslint/lib/config/config-file.js:362:28)
```


I tried to install it by hand and this is what I get then:

```
npm install eslint-config-standard
agile-sdk@0.4.0 /home/dp/git/agile-sdk
├── UNMET PEER DEPENDENCY eslint@3.15.0
├── UNMET PEER DEPENDENCY eslint-config-standard@10.2.1
├── UNMET PEER DEPENDENCY eslint-plugin-import@>=2.2.0
├── UNMET PEER DEPENDENCY eslint-plugin-node@>=4.2.2
├── UNMET PEER DEPENDENCY eslint-plugin-promise@3.4.2
└── UNMET PEER DEPENDENCY eslint-plugin-standard@2.0.1

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN eslint-config-semistandard@7.0.0 requires a peer of eslint-config-standard@^6.0.0 but none was installed.
npm WARN eslint-config-standard@10.2.1 requires a peer of eslint@>=3.19.0 but none was installed.
npm WARN eslint-config-standard@10.2.1 requires a peer of eslint-plugin-import@>=2.2.0 but none was installed.
npm WARN eslint-config-standard@10.2.1 requires a peer of eslint-plugin-node@>=4.2.2 but none was installed.
npm WARN eslint-config-standard@10.2.1 requires a peer of eslint-plugin-promise@>=3.5.0 but none was installed.
npm WARN eslint-config-standard@10.2.1 requires a peer of eslint-plugin-standard@>=3.0.0 but none was installed.
npm ERR! code 1
```
 I am using npm 3.10.10 with node v7.2.1 (nvm)